code/processes/dqc.q

```diff
@@ -188,11 +202,12 @@ reruncheck:{[chkid]
 .servers.CONNECTIONS:`ALL                                                                                       /- set to nothing so that is only connects to discovery
 
 .u.end:{[pt]                                                                                                    /- setting up .u.end for dqe
-  .dqe.endofday[.dqe.dqcdbdir;.dqe.getpartition[];(`results;`configtable);`.dqe];
+  .dqe.endofday[.dqe.dqcdbdir;.dqe.getpartition[];(`results;`configtable);`.dqe;.dqe.tosavedown];
   hdbs:distinct raze exec w from .servers.SERVERS where proctype=`dqcdb;                                        /- get handles for DB's that need to reload
-  .dqe.notifyhdb[1_string .dqe.dqcdbdir]'[hdbs];                                                                /- send message for BD's to reload
-  .timer.removefunc'[exec funcparam from .timer.timer where `.dqe.runcheck in' funcparam];
-  .timer.removefunc'[exec funcparam from .timer.timer where `.u.end in' funcparam];
+  .dqe.notifyhdb[.os.pth .dqe.dqcdbdir]'[hdbs];                                                                 /- send message for BD's to reload
```

Comment:
typo in comment (same line with same typo appears various other places as well)

Reply:
what would be the typo in the comment? I understand that we should use .os instead of the other way but no too sure what you meant by typo in comment

Reply:
I assume `BD's` is meant to be `DB's` (i.e. databases)?